controllers/memberoperatorconfig/memberoperatorconfig_controller.go

```diff
@@ -105,9 +106,30 @@ func (r *Reconciler) handleWebhookDeploy(ctx context.Context, cfg membercfg.Conf
 		if err := deploy.Webhook(ctx, r.Client, r.Client.Scheme(), namespace, webhookImage); err != nil {
 			return err
 		}
+		// TODO --  temporary migration step to delete the webhook with the old names
+		deleted, err := deploy.Delete(ctx, r.Client, r.Client.Scheme(), namespace, true)
+		if err != nil {
+			return err
+		}
+		if deleted {
+			logger.Info("Deleted previously deployed webhook app")
+		}
+		// TODO -- end temporary migration
 		logger.Info("(Re)Deployed users' pods webhook")
 	} else {
-		logger.Info("Skipping deployment of users' pods webhook")
+		memberNS := &corev1.Namespace{}
+		if err := r.Client.Get(ctx, types.NamespacedName{Name: namespace}, memberNS); err != nil {
+			return errs.Wrapf(err, "cannot get member namespace %s", memberNS)
+		}
+		if value, found := memberNS.Labels["toolchain.dev.openshift.com/webhook"]; found && value == "disabled" {
```

Comment:
ok, I now see why you added the label in the e2e tests, but it doesn't answer why we need it here. Disabling of the webhook should be fully managed by ToolchainConfig/MemberConfig CR.

Reply:
good point, I've changed this in https://github.com/codeready-toolchain/member-operator/pull/562/commits/6f50ffab500165de806d03edc0b2b28807fd8d9c